packages/shared/src/store/groupActions.ts

```diff
@@ -564,59 +565,50 @@ export async function moveNavSection(
   const existingGroup = await db.getGroup({ id: group.id });
 
   const navSections = group.navSections ?? [];
-  const sectionIndex = navSections.findIndex(
+  const currentIndex = navSections.findIndex(
```

Comment:
is this `moveNavSection` method dead code in a world with `updateGroupNavigationBatch`? Same question for `moveChannelToNavSection`, `moveChannel`, and `updateNavSectionMeta`

If so, can we remove unused methods at the `action` and `api` layers?